### PR TITLE
adding database

### PR DESCRIPTION
**Pull request by jakkper**

### CODE REVIEW

File: tools/MMS-data_creator.sql
Comment (hubertborysowski) [inline tools/MMS-data_creator.sql:1]: `CREATE TABLE` names are inconsistent. Start with capital or non-capital letter.

File: tools/MMS-data_creator.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE IF NOT EXISTS `mms_database`;
+
+USE `mms_database`;
+
+CREATE TABLE `users` (
+  `email` varchar(50) NOT NULL,
+  `password` varchar(30) NOT NULL,
```
Comment (hubertborysowski) [inline tools/MMS-data_creator.sql:7]: Is password being encrypted?

File: tools/REDME.md
Comment (hubertborysowski) [inline tools/REDME.md:1]: `REDME.md`, typo in name.

File: tools/MMS-data_creator.sql
Comment (hubertborysowski) [inline tools/MMS-data_creator.sql:1]: 2023-07-11 18_21_38-localhost _ 127 0 0 1 _ phpMyAdmin 5 2 1 - Opera
Error while creating database. `MeasurmentUnits`, `ProductType`, `ProductInstance` are not present in database.

**Review by hubertborysowski — CHANGES_REQUESTED** [review]

File: tools/MMS-data_creator.sql (hunk 1 above)
Comment (jakkper) [reply tools/MMS-data_creator.sql:7]: Paswswords are going to be encrypted in API and then send to the database.

File: tools/REDME.md
Comment (Mateusz-Dobrzynski) [reply tools/REDME.md:1]: The contents of this file should be appended to the main README file in the Setup section.

File: tools/MMS-data_creator.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  FOREIGN KEY (measurmentUnitId) REFERENCES measurmentUnits(measurmentUnitId)
+);
+
+CREATE TABLE `productInstance`(
+    `instanceId` INT AUTO_INCREMENT NOT NULL,
+    `productName` varchar(100) NOT NULL,
+    `fridgeId` INT NOT NULL,
+    `productTypeId` INT,
+    `amount` INT,
+    `exporationDate` date,
```
Comment (Mateusz-Dobrzynski) [inline tools/MMS-data_creator.sql:39]: Typo. Should be `expirationDate`

File: tools/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Possible way of making localhosted database.
```
Comment (Mateusz-Dobrzynski) [inline tools/README.md]: Applies for Windows machines

File: README.md [hunk 4]
```diff
@@ -1,2 +1,21 @@
 # meals-management-system
 Meals Management System aims to reduce food waste by suggesting recepies based on products in your fridge and their expiration date
+
+Creating database
```
Comment (Mateusz-Dobrzynski) [inline README.md:4]: You can create a separate section by using `##` as a h2 heading

**Review by Mateusz-Dobrzynski — APPROVED** [review]
In future commits, pay attention to commit messages. Refer to `GUIDELINES.md` for branch naming and commit message conventions.

**Review by hubertborysowski — APPROVED** [review]

**Pull request merged**